### ISSUE TITLE
quantum-transfer-learning /c2q_transfer_learning_ants_bees.ipynb

### ISSUE THREAD

**Issue by mehrnia** [XanaduAI/quantum-transfer-learning#23]
with n_epochs=30 I get accuracy of 0.47. but you mentioned 
"Remark: To reproduce the results of [the paper], num_epochs should be set to 30 which may take many hours. We suggest to first try with num_epochs=1, just to check that everything runs smoothly."

**Comment by CatalinaAlbornoz** [comment]
Hi @mehrnia, the code in this repo is not maintained. I recommend that you go to the [PennyLane demo on Quantum Transfer Learning](https://pennylane.ai/qml/demos/tutorial_quantum_transfer_learning) instead.
